### ISSUE TITLE
A documentation link in error tracing is not working 

### ISSUE THREAD

**Issue by MehdiMJ1** [LonamiWebs/Telethon#3844]
> https://docs.telethon.dev/en/latest/concepts/entities.html

**This link isn't not working,** _the page does not exist .._ 

**Traceback**
```
Traceback (most recent call last):
  File "/app/plugins/bot/manager.py", line 169, in MessageSend
    await Manager.send_message(
  File "/app/.heroku/python/lib/python3.9/site-packages/telethon/client/messages.py", line 810, in send_message
    entity = await self.get_input_entity(entity)
  File "/app/.heroku/python/lib/python3.9/site-packages/telethon/client/users.py", line 466, in get_input_entity
    raise ValueError(
ValueError: Could not find the input entity for PeerUser(user_id=123456789) (PeerUser). Please read https://docs.telethon.dev/en/latest/concepts/entities.html to find out more details.

```

✅ **It should be changed to** https://docs.telethon.dev/en/stable/concepts/entities.html

**Comment by carasino007** [comment]
> > https://docs.telethon.dev/en/latest/concepts/entities.html
> 
> **This link isn't not working,** _the page does not exist .._
> 
> **Traceback**
> 
> ```
> Traceback (most recent call last):
>   File "/app/plugins/bot/manager.py", line 169, in MessageSend
>     await Manager.send_message(
>   File "/app/.heroku/python/lib/python3.9/site-packages/telethon/client/messages.py", line 810, in send_message
>     entity = await self.get_input_entity(entity)
>   File "/app/.heroku/python/lib/python3.9/site-packages/telethon/client/users.py", line 466, in get_input_entity
>     raise ValueError(
> ValueError: Could not find the input entity for PeerUser(user_id=123456789) (PeerUser). Please read https://docs.telethon.dev/en/latest/concepts/entities.html to find out more details.
> ```
> 
> white_check_mark **It should be changed to** https://docs.telethon.dev/en/stable/concepts/entities.html

https://docs.telethon.dev/en/stable/concepts/entities.html

**Comment by Lonami** [comment]
This link has been fixed in the v1.25 version.
